full_build.sh

```diff
@@ -23,7 +23,6 @@ composer install --ansi
 composer install --no-dev --ansi
 
 # early downgrade individual functions
-bin/rector process src/functions -c build/config/config-downgrade.php --ansi --clear-cache
 bin/rector process vendor/symfony/string/Resources/functions.php -c build/config/config-downgrade.php --ansi --clear-cache
```

Comment:
Why do you think this is not needed now?

Reply:
I copy it from latest `.github/workflows/build_scoped_rector.yaml`, the one that removed is `src/functions`, the  `vendor/symfony/string/Resources/functions.php` still in there.

https://github.com/rectorphp/rector-src/blob/cbbe410affb8afcfc3843e6738e882eef18a559d/.github/workflows/build_scoped_rector.yaml#L55-L56

Reply:
oh, I see that `src/functions` is prepended.